targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Public/PlayFabAuthenticationContext.h.ejs

```diff
@@ -13,7 +14,18 @@ class PLAYFABCOMMON_API UPlayFabAuthenticationContext : public UObject
 {
     GENERATED_BODY()
 public:
-
+    UPlayFabAuthenticationContext()
```

Comment:
It would be a really nice thing to have, of course. In some other (earlier) SDKs we didn't have it stored anywhere and hence we didn't have it in the spec and the original PR example. It should not be too difficult to add, though.

---
In reply to: [259177545](https://github.com/PlayFab/SDKGenerator/pull/407#discussion_r259177545) [](ancestors = 259177545)

Reply:
Should the PlayFabId be disabled by the DISABLE_PLAYFABCLIENT_API symbol in the same way that the ClientSessionTicket is (see GetClientSessionTicket in PlayFabAuthenticationContext.h)?